concordium-consensus/src/Concordium/Kontrol/Bakers.hs

```diff
@@ -319,68 +319,69 @@ getSlotBakersP4 ::
     BlockState m ->
     Slot ->
     m FullBakers
-getSlotBakersP4 genData bs slot = do
-    SeedState{epochLength, epoch = blockEpoch} <- getSeedState bs
-    let epochToSlot :: Epoch -> Slot
-        epochToSlot e = fromIntegral e * (epochLength ^. unconditionally)
-    nextPayday <- getPaydayEpoch bs
-    let nextPaydaySlot = epochToSlot nextPayday
+getSlotBakersP4 genData bs slot =
+    getSeedState bs >>= \case
+        SeedStateV0{epochLength, epoch = blockEpoch} -> do
```

Comment:
I have added some documentation that this is not used in V1. The type system checks that `SeedStateV1` is actually impossible because of the constraint `SeedStateVersionFor (MPV m) ~ 'SeedStateVersion0`.